fenix/app/src/androidTest/java/org/mozilla/fenix/onboarding/view/JunoOnboardingMapperTest.kt

```diff
@@ -20,15 +20,27 @@ class JunoOnboardingMapperTest {
         HomeActivityIntentTestRule.withDefaultSettingsOverrides(skipOnboarding = true)
 
     @Test
-    fun showNotificationTrue_pagesToDisplay_returnsSortedListOfAllConvertedPages() {
+    fun showNotificationTrue_showAddWidgetFalse_pagesToDisplay_returnsSortedListOfAllConvertedPages_withoutAddWidgetPage() {
```

Comment:
Could we use our usual GIVEN WHEN THEN test naming structure? Wondering if there was a reason for this?

Reply:
I don't think we are able to use GIVEN WHEN THEN structure with backticks in instrumented tests unless minSdkVersion >= 30.

Reply:
Oh I see we are in `androidTest`. Sorry my bad.